app/settings.py

```diff
@@ -56,6 +63,10 @@ def get_env_or_fail(key):
     "EQ_SUBMISSION_CONFIRMATION_TOPIC_ID", "eq-submission-confirmation-topic"
 )
 EQ_INDIVIDUAL_RESPONSE_LIMIT = int(os.getenv("EQ_INDIVIDUAL_RESPONSE_LIMIT", "1"))
+EQ_INDIVIDUAL_RESPONSE_POSTAL_DEADLINE = utcoffset_or_fail(
```

Comment:
We should document new env vars in the README. We have missed a few, however, I am not against avoiding adding Census specific env vars in README.